### DISCUSSION TITLE
2D in 3D environment?

### DISCUSSION THREAD

**Discussion by Therealjosephchrzempiec** [networked-aframe/networked-aframe#399] [Q&A]
Hello, I was curious. if it was possible to do a 2D movement within 3D? What I mean by that is to move the head avatar up,down,left or right within a location? Something like this image.

![2din3d](https://user-images.githubusercontent.com/50338148/216736156-8323406d-c31f-4aa0-a16e-c7172bf253b2.png)

i'm working on a chat that has 2D within 3D environment.

Joseph

**Comment by vincentfretin** [comment]
Hello, you can modify the position and rotation components. If you want a 2D movement, don't change the y rotation and z position I guess.

**Comment by Therealjosephchrzempiec** [comment]
Well in 2d its X and Y position up and down. Problem is I have no clue how I can do that in networked-aframe.

**Reply by vincentfretin** [reply]
#404 may be relevant to you.
